test/ui-test/extensionInstallationUITest.ts

```diff
@@ -0,0 +1,63 @@
+import { fail } from 'assert';
+import { expect } from 'chai';
+import { ActivityBar, ExtensionsViewItem, ExtensionsViewSection, VSBrowser, WebDriver } from 'vscode-extension-tester';
+import { KNativeConstants } from './common/constants';
+
+/**
+ * @author Ondrej Dockal <odockal@redhat.com>
+ */
+export function extensionsUITest(): void {
+  let driver: WebDriver;
+
+  before(() => {
+    driver = VSBrowser.instance.driver;
+  });
+
+  describe('Knative extension', () => {
+    it('should be installed among extensions', async function context() {
+      this.timeout(10000);
+      // eslint-disable-next-line @typescript-eslint/await-thenable
+      const viewBar = await new ActivityBar().getViewControl('Extensions');
+      const sideBar = await viewBar.openView();
+      const section = (await sideBar.getContent().getSection('Installed')) as ExtensionsViewSection;
+      const item = await driver.wait(async () => section.findItem(`@installed ${KNativeConstants.KNATIVE_EXTENSION_NAME}`), 3000);
+      expect(item).to.be.an.instanceOf(ExtensionsViewItem);
+      expect(await item.getTitle()).to.equal(KNativeConstants.KNATIVE_EXTENSION_NAME);
+    });
+
+    it('Activity Bar should be available', async function context() {
+      this.timeout(5000);
+      try {
```

Comment:
Where is the `expect` that the test must pass?

Reply:
Assert here is implicated by not failing the test in catch block.

Reply:
So the `fail` will register with the test framework?

Reply:
Hm, it should be this way. Always was, but I did not check :D

Reply:
do we need a test for the test? ;-)

Reply:
I really hope not! :D